docs/cne-install.md

```diff
@@ -0,0 +1,177 @@
+# SAT Upgrade with CNE Installer
+
+## Upgrade the System Admin Toolkit Product Stream
+
+Describes how to upgrade the System Admin Toolkit (SAT) product
+stream by using the Compute Node Environment (CNE) installer (`cne-install`).
+
+The CNE installer can only upgrade the SAT product stream for this release. See [Install the System Admin Toolkit Product Stream](install.md) for installation instructions.
+
+Upgrading SAT with `cne-install` is recommended because the process is both automated and logged to help you save time. The CNE installer can be used to upgrade SAT alone or with other supported products. Refer to the [*HPE Cray EX System Software Getting Started Guide (S-8000)*](https://www.hpe.com/support/ex-S-8000) for detailed information about `cne-install` and its options.
+
+### Prerequisites
+
+- CSM is installed and verified.
+- cray-product-catalog is running.
+- There must be at least 2 gigabytes of free space on the manager NCN on which the
+  procedure is run.
+
+### Notes on the Procedures
+
+- Ellipses (`...`) in shell output indicate omitted lines.
+- In the examples below, replace `x.y.z` with the version of the SAT product stream
+  being upgraded.
+- 'manager' and 'master' are used interchangeably in the steps below.
+
+### Pre-Upgrade Procedure
+
+1.  Start a typescript and set the shell prompt.
+
+    The typescript will record the commands and the output from this upgrade.
+    The prompt is set to include the date and time.
+
+    ```screen
+    ncn-m001# script -af product-sat.$(date +%Y-%m-%d).txt
+    ncn-m001# export PS1='\u@\H \D{%Y-%m-%d} \t \w # '
+    ```
+
+### Upgrade Procedure
+
+1.  Copy the release distribution gzipped tar file to `ncn-m001`.
+
+    Whether you are upgrading SAT alone or with other supported products, copy the file in the same media directory as all other supported products.
+
+1.  Run the CNE installer.
+
+    - If you are upgrading SAT along with other supported products, run the following command.
+
+        ```screen
+        ncn-m001# cne-install -m MEDIA_DIR install -B WORKING_BRANCH -bpc BOOTPREP_CONFIG_CN -bpn BOOTPREP_CONFIG_NCN
+        ```
+
+        The `cne-install` command will use the provided `BOOTPREP_CONFIG_CN` and `BOOTPREP_CONFIG_NCN` files for the run.
+
+    - If you are upgrading SAT alone, run the following command.
+
+        ```screen
+        ncn-m001# cne-install -m MEDIA_DIR install -B WORKING_BRANCH
+        ```
+
+1.  **Optional:** Stop the typescript.
+
+    **NOTE**: This step can be skipped if you wish to use the same typescript
+    for the remainder of the SAT upgrade. See [Next Steps](#next-steps).
+
+    ```screen
+    ncn-m001# exit
+    ```
```

Comment:
Suggested spacing for this section:
~~~suggestion
1. Run the CNE installer.

   - If you are upgrading SAT along with other supported products, run the following command.

     ```screen
     ncn-m001# cne-install -m MEDIA_DIR install -B WORKING_BRANCH -bpc BOOTPREP_CONFIG_CN -bpn BOOTPREP_CONFIG_NCN
     ```

     The `cne-install` command will use the provided `BOOTPREP_CONFIG_CN` and
     `BOOTPREP_CONFIG_NCN` files for the run.

   - If you are upgrading SAT alone, run the following command.

     ```screen
     ncn-m001# cne-install -m MEDIA_DIR install -B WORKING_BRANCH
     ```

1. **Optional:** Stop the typescript.

   **NOTE**: This step can be skipped if you wish to use the same typescript
   for the remainder of the SAT upgrade. See [Next Steps](#next-steps).

   ```screen
   ncn-m001# exit
   ```
~~~